mock-abis/src/main/java/io/mosip/proxy/abis/service/impl/ProxyAbisInsertServiceImpl.java

```diff
@@ -536,13 +523,12 @@ private IdentityResponse.Analytics getAnalytics() {
 		return a;
 	}
 
-
-	public  String saveUploadedFileWithParameters(MultipartFile uploadedFile, String alias,
-												  String password, String keystore) {
+	public String saveUploadedFileWithParameters(MultipartFile uploadedFile, String alias, String password,
+			String keystore) {
 		try {
 			logger.info("Uploading certificate");
 			byte[] bytes = uploadedFile.getBytes();
-			Path path = Paths.get(UPLOAD_FOLDER + "/"+ uploadedFile.getOriginalFilename());
+			Path path = Paths.get(UPLOAD_FOLDER + "/" + uploadedFile.getOriginalFilename());
```

Comment:
Do not use the the +

Reply:
Path path = Paths.get(keystoreFilePath.toString(), uploadedFile.getOriginalFilename());